utils/_context/_scenarios/parametric.py

```diff
@@ -469,6 +469,7 @@ def dotnet_library_factory():
 ENV CORECLR_ENABLE_PROFILING=0
 ENV CORECLR_PROFILER={{846F5F1C-F9AE-4B07-969E-05C26BC060D8}}
 ENV CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
+ENV LD_PRELOAD=/opt/datadog/continuousprofiler/Datadog.Linux.ApiWrapper.x64.so
```

Comment:
Is it ok to add this to _all_ parametric tests, or should we add it to the new tests only?

Reply:
I think it's better to add it to all tests. My reasoning is that it _shouldn't_ impact the other tests, so that's a good way to make sure it doesn't.